src/rev_ai/models/job_options.py

```diff
@@ -0,0 +1,14 @@
+class JobSubmitOptions:
+	def __init__(
+		self, 
+		metadata=None, 
+		callback_url=None, 
+		skip_diarization=None
+	):
+		"""
+		:param metadata: info to associate with the transcription job
+        :param callback_url: callback url to invoke on job completion as a webhook
+        """
+		self.metadata = "" if metadata == None else metadata
+		self.callback_url = callback_url
+		self.skip_diarization = False if skip_diarization == None else skip_diarization
```

Comment:
this is a bit weird for me, shouldn't this be a `bool`? In which case the right hand expression should simply be `skip_diarization is not None`.